src/ui/assets/styles/components/_discite.scss

```diff
@@ -38,6 +38,12 @@ $module: 'discite';
 		@include media(desktop) {
 			padding-right: 300px;
 		}
+
+		@include m(module) {
```

Comment:
If you need to you can use discite__counter and add the modifier `@include m(module)` in there, and get rid of the nested modifier

Reply:
done